server/service/src/sync/test/integration/central_server_configurations.rs

```diff
@@ -111,7 +111,7 @@ impl NewSiteProperties {
             store_id: uuid(),
             name_id: uuid(),
             // TODO max that can be used ?
-            site_id: thread_rng().gen::<u16>(),
+            site_id: thread_rng().gen::<i32>(),
```

Comment:
Technically this is correct, but I believe the old `u16` was trying to limit the values to positive values of `i32`. But, `get_u16(KeyValueType)` doesn't exist in `KeyValueStoreRepository` so I just changed this rather than define that.

The only illegal value is `0` and any sites that may already exist on the central server.

Reply:
Hmm, i had issue, might have been with u32, so I guess i32 should work since it's long integer in 4d